infoschema/slow_log_test.go

```diff
@@ -0,0 +1,61 @@
+package infoschema
+
+import (
+	"bufio"
+	"bytes"
+	"testing"
+
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/types"
+)
+
+func TestParseSlowLogFile(t *testing.T) {
+	slowLog := bytes.NewBufferString(
+		`# Time: 2019-01-24-22:32:29.313255 +0800
+# Txn_start_ts: 405888132465033227
+# Query_time: 0.216905
+# Process_time: 0.021 Request_count: 1 Total_keys: 637 Processed_keys: 436
+# Is_internal: true
+select * from t;`)
```

Comment:
do we have potential that one record be split into two log file by logrotate?  I'm not confirm that :D, if so maybe we need a testcase for that.

Reply:
Maybe, but currently will only parse 1 file. So if the record was split into 2 file. The record will not present to the result.

Reply:
I will add a TODO to support parse multiple log file.